lambda/shared/src/test/scala-2/feral/lambda/events/SnsEventSuite.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda.events
+
+import io.circe.Json
+import io.circe.literal._
+import munit.FunSuite
+import scodec.bits.ByteVector
+
+import java.time.Instant
+import java.util.UUID
+
+class SnsEventSuite extends FunSuite {
+  import SnsEventSuite._
+
+  test("decoder") {
+    assert(clue(event.as[SnsEvent].toTry.get) == clue(decoded))
```

Comment:
Nit and/or question :) I've never used `clue` before so TIL, but I'd guess that `assertEquals` does effectively the same thing here?

```suggestion
    assertEquals(event.as[SnsEvent].toTry.get, decoded)
```